framework/apache-superset/test.py

```diff
@@ -83,7 +83,7 @@ def test_ui():
         page.get_by_role("button", name="Run").click()
         page.wait_for_timeout(500)
         page_body = page.text_content("div.ant-tabs-content-holder")
-        assert "42 rows returned" in page_body
+        assert "42 rows" in page_body
```

Comment:
That slipped in accidentally. After rebasing correctly, it should go away.

Reply:
I am not sure which commit could have caused this, absolutely didn't touch anything on superset...

Reply:
This is the relevant commit. You can just delete it from your branch by rebasing interactively. Let me know if you need further guidance about it. Otherwise, while I don't know how GitHub manages it, it will probably automatically go away when rebase-merging the PR, because, well, the change is already in `main`.

![image](https://github.com/crate/cratedb-examples/assets/453543/7b4f4261-140d-451d-a7ad-0442145e4e47)

Reply:
It went away with rebase. Thanks!